core/metadata-3.2.md

```diff
@@ -41,6 +59,16 @@ Implementations MAY provide a mechanism for limiting visibility of certain
 metadata to certain users. Such mechanism is implementation-defined;
 for instance, it may depend on some permission level or a flag.
 
+### Capability value
+
+The value of the `metadata` capability introduced in this document MUST
+have the following format:
+
+    [<anything>,]maxsub=<N>[,<anything>]`
```

Comment:
the `<anything>` feels really sketchy. Can we instead specify this the same way we specify tokens in the STS spec [here](https://ircv3.net/specs/extensions/sts.html#details)?